Automattic/wp-calypso
# Help: Scroll to Top of the Page on Loading

#### Changes proposed in this Pull Request

* Scroll to the top of the page when loading the Help section 

#### Testing instructions

* Visit the Customer Home
* Scroll down
* Click "More help" under the "Get Help" card
* Verify that when going to the Help section, the top of the page appears 

<img width="388" alt="Screenshot 2021-04-13 at 21 20 46" src="https://user-images.githubusercontent.com/43215253/114615895-21287f00-9c9e-11eb-9e1e-6709cd05c6be.png">

cc @sixhours 

Fixes #51912
